tensorflow/compiler/xla/service/gpu/BUILD

```diff
@@ -964,6 +966,8 @@ cc_library(
         "//tensorflow/core:lib_proto_parsing",
         "//tensorflow/core:tflite_portable_logging",
         "//tensorflow/core/platform:stream_executor_no_cuda",
+        "//tensorflow/core/kernels:gpu_utils",
```

Comment:
We can't introduce XLA->TF dependency edges. The corresponding code can be moved to SE.